frontend/src/app/shared/shared.module.ts

```diff
@@ -437,5 +437,6 @@ export class SharedModule {
     library.addIcons(faFaucetDrip);
     library.addIcons(faTimeline);
     library.addIcons(faCircleXmark);
+    library.addIcons(faCalendarCheck);
```

Comment:
I sneak this change here for another PR pending merge in mempool.space repo. Not worth making a separate PR 😄.